project/profiles/capacity4more/capacity4more.profile

```diff
@@ -149,8 +148,19 @@ function capacity4more_setup_set_og_permissions() {
   og_role_grant_permissions($auth_rid, $permissions);
 
   // Set OG_ADMINISTRATOR_ROLE permissions.
-  $permissions = array();
+  $content_types = array(
+    'wiki_page',
+  );
 
+  $permissions = array();
+  foreach ($content_types as $content_type) {
+    $permissions = array_merge($permissions, array(
+      "create $content_type content",
+      "update own $content_type content",
+      "delete own $content_type content",
```

Comment:
@ybaras 
Please remove the _delete_ permission since this is not part of the request.

![selection_212](https://cloud.githubusercontent.com/assets/5985683/16545545/b67140ce-4136-11e6-98d7-721824fac05a.png)